corehq/toggles.py

```diff
@@ -1,3 +1,32 @@
+"""
+    Toggles
+
+    Toggles, also known as feature flags, allow limiting access to a set of functionality.
+
+    Most toggles are configured by manually adding individual users or domains in the Feature Flags
+    admin UI. These are defined by adding a new `StaticToggle` in this file. See `PredictablyRandomToggle`
+    and `DynamicallyPredictablyRandomToggle` if you need a toggle to be defined for a random subset
+    of users.
+
+    Namespaces define the type of access granted. NAMESPACE_DOMAIN allows the toggle to be enabled
+    for individual project spaces. NAMESPACE_USER allows the toggle to be enabled for individual users,
+    with the functionality visible to only that user but on any project space they visit.
+    NAMESPACE_DOMAIN is preferred for most flags, because it can be confusing for different users
+    to experience different behavior. Domain-based flags are like a lightweight privilege that's
+    independent of a software plan. User-based flags are more like a lightweight permission that's
+    independent of user roles.
+
+    Tags define how a feature is supported. See descriptions below. When in doubt, use TAG_CUSTOM
```

Comment:
Can you elaborate on what supported means. Does that mean what team supports it? 

Also, I was curious if tags affected behavior or if it's just for information/labeling.